src/monorepo-utils/src/Workspaces.js

```diff
@@ -69,4 +70,17 @@ module.exports = {
     });
     return packageJSONLocations;
   },
+
+  getWorkspaceLocationMap(): $ReadOnlyMap<string, string> {
```

Comment:
@tbergq Why isn't the `getWorkspacesSync` good enough for your use-case? The functions which are already available here seems to be covering your case as well. 🤔

Reply:
`getWorkspacesSync ` only gives me the path to each package.json file. I would have to read every json file and check the name.

I am all for using existing functionality, I just can't find anything that fits the use case well

Reply:
@mrtnzlml please check again